pyproject.toml

```diff
@@ -3,21 +3,19 @@ requires = ["setuptools", "setuptools-scm"]
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "timelapse-colorizer-data"
+name = "colorizer-data"
 authors = [
     {name = "", email = "danielt@alleninstitute.org"},
+    {name = "", email = "peyton.lee@alleninstitute.org"},
 ]
-description = "data prep scripts"
-# readme = ""
+description = "Utilities to convert data for viewing in Timelapse Colorizer"
+readme = "README.md"
```

Comment:
I think you can add bump-my-version as a dev dependency in here

Reply:
I'm going to leave it out of the dependencies, because it'll require anyone who installs `colorizer-data` to also install it. Scientists won't really need it, and any devs who want to contribute should already be reading our README and following the install step there, I hope.

Reply:
The way to formalize it is like this:
```
[project.optional-dependencies]
# development dependency groups
dev = [
    'bump-my-version',
]
```
Then a contributor can do `pip install -e .[dev]`

Reply:
But I know it's only one single package so no big deal yet